### ISSUE TITLE
Majinbuu dependencie

### ISSUE THREAD

**Issue by alexandre-mouton-brady** [WebReflection/hyperHTML#143]
Hey, I really love your library and I've been following the changes in the past few weeks.

Since you reverted back to using Majinbuu, I wonder if that shouldn't be listed in the dependencies back too?

Sorry if that's a silly question.

**Comment by WebReflection** [comment]
Yes, it probably should, sorry

**Comment by alexandre-mouton-brady** [comment]
Don't be sorry, you're doing an amazing job and I'm glad I could "help" :)

**Comment by WebReflection** [comment]
I've rushed too many patches lately after freaking out for something I've measured wrong.

majinbuu is a "fake" dependency needed only on build time, it's indeed loaded via unpkg.cdn directly as regular ESM module.

I am planning to release hyperHTML with a dumb/simple engine and make majinbuu optional but since I don't want to rush patches anymore I am probaly keeping this package as it is.

TL;DR this does not affect the usage of the library, but I'll keep this bug open until everything is properly solved.

Thanks again for the repo
